tests/neo.UnitTests/Ledger/UT_PoolItem.cs

```diff
@@ -96,7 +96,7 @@ public Transaction GenerateTxWithFirstByteOfHashGreaterThanOrEqualTo(byte firstH
             do
             {
                 tx = GenerateTx(networkFee, size);
-            } while (tx.Hash >= new UInt256(TestUtils.GetByteArray(32, firstHashByte)));
+            } while (tx.Hash < new UInt256(TestUtils.GetByteArray(32, firstHashByte)));
 
```

Comment:
@eryeer why you need to change the UT?

Reply:
Because I find the logic have some problem in this place, if we want to create a tx whose hash is greater than of equal to the input value, we need to keep the recursion until tx hash is not smaller than the input value, right?

Reply:
But it works without the change too, right?

Reply:
No, the expect result in assertion is incorrect as well. 😂

Reply:
The previous expected result is that the transactions are arranged in ascending order according to the hash value, while the transactions are actually arranged in descending order according to the hash value. So I updated the assertion as well.